.github/workflows/test_pr.yml

```diff
@@ -34,4 +34,29 @@ jobs:
         with:
           args: deploy -P roster-pr --only hosting:project-dashboard-pull-request
         env:
-          FIREBASE_TOKEN: ${{ secrets.FIREBASE_TOKEN }}
\ No newline at end of file
+          FIREBASE_TOKEN: ${{ secrets.FIREBASE_TOKEN }}
+
+
+
+run: npm run test -- --coverage
```

Comment:
No need to run these steps again. Also run statement should be inside a step.

New step would be extracting the coverage total % and pushing that to env. Which will be then used in the "message" part of "Create the Badge" step